profile/rwahs.xml

```diff
@@ -304,55 +304,13 @@
         </label>
       </labels>
       <items>
-        <item idno="subject" enabled="1" default="0">
+        <item idno="Subject" enabled="1" default="0">
```

Comment:
I wasn't sure whether to capitalise this, but I did, which meant more changes elsewhere in the profile.